app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] as part of the requirements, kindly add the  `foreign_key: 'author_id'` to the user's associations 🙏